apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/GetQueryResultsOptions.cs

```diff
@@ -56,7 +56,20 @@ public sealed class GetQueryResultsOptions
         /// </summary>
         public string PageToken { get; set; }
 
-        internal void ModifyRequest(GetQueryResultsRequest request)
+        /// <summary>
+        /// Whether to use 64-bit integers for timestamps, instead of
+        /// floating point, in order to preserve precision. If not set, this is effectively true.
+        /// </summary>
+        public bool? UseInt64Timestamp { get; set; }
```

Comment:
I think I'd do this instead of moving away from the usual pattern.

```suggestion
        /// <summary>
        /// Whether to use 64-bit integers for timestamps, instead of
        /// floating point, in order to preserve precision. Defaults to true.
        /// </summary>
        public bool UseInt64Timestamp { get; set; } = true;
```

And then anywhere in our public surface that we can recieve these options, do `options ?? new GetQueryResultsOptions()` or `options ?? GetQueryResultsOptions.Default`.

Reply:
The normal pattern for this project is to use a nullable type - I *think* we've done that for every option. It means that the options are composable. Not that we end up actually using that ability at the moment, but it's why we have it.

We could still use the "have a default set of options and move ModifyRequest back to being an instance method though", if you want.

Reply:
Yep, it's eaither we brake one pattern or the other. But yes, composability would be harder to achieve if we make some options non-nullable. So yes, this is better in the long run.